tests/playwright/specs/shared/helpdesk/home_config.spec.ts

```diff
@@ -0,0 +1,235 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+import { test, expect } from '../../../fixtures/authenticated';
+import { HomeConfigPage } from '../../../pages/helpdesk/HomeConfigPage';
+import { GlpiApi } from '../../../utils/GlpiApi';
+import { SessionManager } from '../../../utils/SessionManager';
+
+let config_page: HomeConfigPage;
+let profile_id: number;
+
+test.beforeEach(async ({ page, request }) => {
+    // Load helpdesk profile
+    const session = new SessionManager(request);
+    await session.changeProfile("Super-Admin");
+
+    // Create a test profile with a few tiles
+    const glpi_api = new GlpiApi(test.info().parallelIndex);
+    profile_id = await glpi_api.createItem('Profile', {
+        name: 'Helpdesk profile for e2e tests',
+        interface: 'helpdesk',
+    });
+    [
+        {
+            title: "Browse help articles",
+            description: "See all available help articles and our FAQ.",
+            illustration: "browse-kb",
+            page: "faq",
+        },
+        {
+            title: "Request a service",
+            description: "Ask for a service to be provided by our team.",
+            illustration: "request-service",
+            page: "service_catalog ",
+        },
+        {
+            title: "Make a reservation",
+            description: "Pick an available asset and reserve it for a given date.",
+            illustration: "reservation",
+            page: "reservation",
+        },
+        {
+            title: "View approval requests",
+            description: "View all tickets waiting for your validation.",
+            illustration: "approve-requests",
+            page: "approval",
+        }
+    ].forEach(async (tile, i) => {
+        const tile_id = await glpi_api.createItem('Glpi\\Helpdesk\\Tile\\GlpiPageTile', tile);
+        await glpi_api.createItem('Glpi\\Helpdesk\\Tile\\Profile_Tile', {
+            profiles_id: profile_id,
+            itemtype: 'Glpi\\Helpdesk\\Tile\\GlpiPageTile',
+            items_id: tile_id,
+            rank: i,
+        });
+    });
+
+    // Load POM object and go to page
+    config_page = new HomeConfigPage(page);
+    await config_page.goto(profile_id);
+    await config_page.fixHtmlSortableRoles();
+});
+
+test('can remove tiles', async ({ page }) => {
+    // Make sure the tile exist before trying to delete it
+    await expect(config_page.getTile("Request a service")).toBeVisible();
```

Comment:
Not a fan that we trade command queuing for "await hell". At the very least, we should be careful to not use await for every single line of playwright test code. Should use `Promise.all` when possible to run things together at least.

Reply:
IMO (and most online opinions from what I can tell) `await` is correct modern way of using javacript while cypress command queue is an un-intuitive nightmare that lead to callback hell.